components/tests/ui/testcases/web/webadmin_create_group_and_user.txt

```diff
@@ -142,9 +145,8 @@ Check Admin Edit Self
     Go To                             ${USERS URL}
     Click Element                     xpath=//table[@id="experimenterTable"]/tbody/tr[descendant::td[contains(text(), '${ROOT FULL NAME}')]]//a[contains(@class, "btn_edit")] 
 
-    Wait Until Page Contains Element  id=id_administrator       ${WAIT}
-    Checkbox Should Be selected       id=id_administrator
-    Element Should Be Disabled        id=id_administrator
+    Wait Until Page Contains Element  xpath=//input[@type="radio" and @value="administrator"]       ${WAIT}
+    Radio Button Should Be Set To     role  administrator
```

Comment:
odd double space, is it usual for these? I figured you were vertically lining up with something but I can't see what